frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/task-sidebar/TaskSidebar.tsx

```diff
@@ -135,6 +140,34 @@ const TaskSidebar: React.FC<TaskSidebarProps> = (props) => {
           </>
         )}
 
+        {workspaces && (
```

Comment:
the workspaces section on the task sidebar looks like this

![ts](https://user-images.githubusercontent.com/22490998/108851810-d8900600-760a-11eb-80d0-458a47bc2abd.png)

better to check the length of the array
```suggestion
        {workspaces.length !== 0  && (
```

Reply:
Updated.